lms/views/helpers/_via.py

```diff
@@ -37,6 +37,7 @@ def via_url(request, document_url):
     ]
     query_string_as_list.append(("via.open_sidebar", "1"))
     query_string_as_list.append(("via.request_config_from_frame", request.host_url))
+    query_string_as_list.append(("via.config_frame_ancestor_level", 2))
```

Comment:
Ah, I didn't know that integer values would be automatically converted to strings, although that's not too surprising.